src/services/document_service.py

```diff
@@ -116,12 +121,15 @@ def save(
         path="",
         update_uncontained: bool = False,
         combined_document_meta: dict | None = None,
+        initial: bool = False,  # Should never be passed to children
```

Comment:
I don't understand this param. Perhaps add more info to the docstring?

Reply:
more description added in https://github.com/equinor/data-modelling-storage-service/pull/435